### ISSUE TITLE
Add automatic validation set to the Drawing Classifier

### ISSUE THREAD

**Issue by shantanuchhabra** [apple/turicreate#1493]
To prevent overfitting, we should implement a validation split as part of training in the Drawing Classifier.
